src/client/marble-manager.js

```diff
@@ -101,6 +101,7 @@ const MarbleMesh = function(marbleData) {
 	let nameSpriteOptions = {};
 	if (_userData && _userData.username === this.name) {
 		nameSpriteOptions.color = "#BA0069";
+		nameSpriteOptions.drawOnTop = 9e9;
```

Comment:
I like that you made this a variable in the `nameSpriteOptions`, but the variable name does not really represent what it actually does. I would suggest renaming this to `nameSpriteOptions.renderOrder`